### PR TITLE
Minor post-first-release-cleanup

### PR DESCRIPTION
**Pull request by massdosage**
- added travis file (might need some tweaking)
- added badges and license info to readme
- updated changelog with 0.0.1 release

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: java
+jdk:
+  - openjdk7
+os:
+  - linux
+install:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -P travis
+script:
```
Comment (courtsVII) [inline .travis.yml]: skip tests?

Comment (courtsVII) [reply .travis.yml]: Ah I see thats standard, never mind

**Comment by ddcprg** [comment]
Looks good, can we add branch protection to this project?

**Comment by massdosage** [comment]
Enabled some branch protections

**Pull request merged**